src/main/kotlin/de/bund/digitalservice/useid/events/EventController.kt

```diff
@@ -0,0 +1,55 @@
+package de.bund.digitalservice.useid.events
+
+import mu.KotlinLogging
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.codec.ServerSentEvent
+import org.springframework.web.bind.annotation.CrossOrigin
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Flux
+import reactor.core.publisher.FluxSink
+import reactor.core.publisher.Mono
+
+@RestController
+@RequestMapping("/api/v1")
+class EventController(eventHandler: EventHandler) {
+    private val log = KotlinLogging.logger {}
+
+    private val eventHandler: EventHandler
+
+    init {
+        this.eventHandler = eventHandler
+    }
+
+    /**
+     * This endpoint receives events from the eID client (i.e. Ident-App) and publishes them to the respective consumer.
+     */
+    @PostMapping("/events")
+    @ResponseStatus(HttpStatus.CREATED)
+    fun send(@RequestBody event: Event): Mono<Event> {
+        log.info { "Received event for consumer: ${event.widgetSessionId}" }
+        eventHandler.publish(event)
+        return Mono.just(event)
```

Comment:
As far as I know, client (browser) won't consume any response, right? I think we can just return`Mono.empty()` instead? what do you think?

Reply:
This endpoint is called by the mobile app. It should consume the response, so that we can return errors if some happen, but in the happy case, we don't need to return anything probably.

Reply:
True, I will return empty for now.

Reply:
Yes, semantically that seems more appropriate for a 201. Thinking about this, a 202 ACCEPTED might be more correct technically. We can only say for sure that we retrieved the request and deal with it.

Reply:
> This endpoint is called by the mobile app. It should consume the response, so that we can return errors if some happen, but in the happy case, we don't need to return anything probably.

@code28 you are right! I think we could also return nothing as payload in case of error and a proper status code (4xx, 5xx), then the client can decide how to handle the status code, e.g. the client can load the error message from translation.. but this idea is probably not for now but future improvement.

Reply:
The endpoint returns status 202 ACCEPTED with an empty Mono now.